scripts/dashboard/install-kcli-deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sudo dnf -y install libvirt libvirt-daemon-driver-qemu qemu-kvm
+sudo usermod -aG qemu,libvirt $(id -un)
+newgrp libvirt
+systemctl enable --now libvirtd
+
+sudo dnf install -y podman
+
+mkdir -p ~/.ssh
+cp -R src/pybind/mgr/dashboard/ci/cephadm/.kcli ~
+
+echo "
+alias kcli='podman run --rm --net host --security-opt label=disable \
```

Comment:
what about the  "--it" parameter?

Reply:
With that I cannot capture the output that I need later on:
```
if [[ -z "${CYPRESS_BASE_URL}" ]]; then
    CYPRESS_BASE_URL="https://$(kcli info vm ceph-node-00 -f ip -v):8443"
fi
```